readthedocs/docsitalia/utils.py

```diff
@@ -66,3 +71,33 @@ def get_projects_with_builds(only_public=True):
     return Project.objects.filter(
         pk__in=filtered_projects
     )
+
+
+def get_international_version_slug(project, lang_slug, version_slug):
+    """For readability of urls we are changing slug names if language is not Italian."""
+    try:
+        lang_slug = lang_slug or project.language
+        version_slug = version_slug or project.get_default_version()
+        if lang_slug != LANG_IT:
+            if version_slug == STABLE:
+                return settings.RTD_STABLE_EN
+            if version_slug == LATEST:
+                return settings.RTD_LATEST_EN
+    except AttributeError:
+        # this means custom versions are not set and we fallback to unchanged behavior
+        pass
+    return version_slug
+
+
+def get_real_version_slug(lang_slug, version_slug):
```

Comment:
Scusate è una cosetta, ma non riusciamo a trovare un nome più chiaro di `get_real_version_slug`? Così non risulta chiarissima la differenza tra le due funzioni.

Reply:
`get_local_version_slug` o `get_italian_version_slug`?

Reply:
ma in questo metedo noi troviamo un slug vero che sta in DB, quindi `get_real_version_slug` non e' cosi male.

Reply:
Va bene `get_real_version_slug`, ora mi è più chiaro.

Reply:
@Amanit forse bisogna avere doc string più chiare: soprattutto che spieghino il ruolo delle due funzioni

Reply:
@yakky fatto

Reply:
@francescozaia è più chiaro anche dalla docstring?